js/script.js

```diff
@@ -0,0 +1,12 @@
+function drawTree(height) {
+  for(var i = 1; i <= height; i++) {
+    var space= "                   " ;
```

Comment:
from where you know how many spaces add here?

Reply:
I fix it